### PR TITLE
Add Codacy integration info

### PR DESCRIPTION
**Pull request by machadoit**
Codacy provides an out-of-the-box solution for automated code review that uses RuboCop

### CODE REVIEW

File: manual/integration_with_other_tools.md [hunk 1]
```diff
@@ -106,3 +106,7 @@ RuboCop::RakeTask.new(:rubocop) do |task|
   task.fail_on_error = false
 end
 ```
+
+## Codacy integration
+
+Although it is not an IDE, with [Codacy](https://www.codacy.com/) you have Rubocop analysis out-of-the-box, and it is free for open source projects. The [codacy-rubocop](https://github.com/codacy/codacy-rubocop) docker runs Rubocop and return its results to Codacy.
```
Comment (pocke) [inline manual/integration_with_other_tools.md]: `s/Rubocop/RuboCop/g`

**Comment by bbatsov** [comment]
Probably there should be a separate section for tools like Codacy. HoundCI and CodeClimate also use RuboCop internally for Ruby projects.

File: manual/integration_with_other_tools.md (hunk 1 above)
Comment (machadoit) [inline manual/integration_with_other_tools.md]: 👍

**Comment by machadoit** [comment]
@bbatsov sounds good also, let me know if I need to do anything else, or should wait for that new section.

**Comment by bbatsov** [comment]
Guess you can create the new section and mention there HoundCI and CodeClimate as well.

**Comment by machadoit** [comment]
@bbatsov I am going to create this new section thx, I will call it "Automated Review" or you guys have any other suggestion?

**Comment by machadoit** [comment]
@bbatsov @pocke I've added the Automated Review section, let me know what you guys think

File: manual/automated_review.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Automated Review
```
Comment (bbatsov) [inline manual/automated_review.md]: Let's name the section "automated code review".

Comment (machadoit) [inline manual/automated_review.md]: thx, done

**Pull request merged**